test/testenv/util.go

```diff
@@ -886,3 +886,16 @@ func DeleteOperatorPod(testcaseEnvInst *TestCaseEnv) error {
 	}
 	return nil
 }
+
+// DeleteFilesOnOperatorPod Delete files on Operator Pod
+func DeleteFilesOnOperatorPod(ctx context.Context, deployment *Deployment, podName string, filenames []string) error {
+	for _, filepath := range filenames {
+		cmd := fmt.Sprintf("rm -f %s", filepath)
+		_, err := ExecuteCommandOnOperatorPod(ctx, deployment, podName, cmd)
```

Comment:
As we discussed offline we might see flakiness if we try to delete files which are not present.

Reply:
i have used the largest app we have for the very same reason. It takes a while for the app to be copied. I tried running the test case multiple times. Got the timing right each time